src/mlmc/quantity.py

```diff
@@ -462,7 +463,7 @@ def _result_qtype(method, quantities):
         :param method: ufunc function
         :return: QType
         """
-        chunks_quantity_level = [q.samples(ChunkSpec(level_id=0, chunk_id=0, n_samples=10)) for q in quantities]
+        chunks_quantity_level = [q.samples(level_id=0, chunk_id=0, n_samples=10) for q in quantities]
```

Comment:
We should use Storage.chunks(..) and single chunk iteration here.

Reply:
Ok. On the other hand, I would like to keep the ability to load data directly without `Storage.chunks`

For example, to obtain data for a histogram:
`self._quantity.samples(level_id=level_id, n_samples=self._sample_storage.get_n_collected()[level_id])`